test.lua

```diff
@@ -4209,6 +4209,40 @@ function nntest.LookupTable()
    end
    local err = padw_sum - padw:sum()
    mytester:assertlt(err,precision, 'padding update error ')
+   -- test whether the weights changes accordingly when maxNorm is not nil
+   local all_index = torch.randperm(totalIndex):int()
+   -- input can have duplicates
+   local input = torch.repeatTensor(all_index:clone():narrow(1,1,nIndex), 2):int()
+   local maxNorm = math.random()
+   local normValue = torch.Tensor{1,2,math.random()}
```

Comment:
This can just be a table instead of a Tensor and then you can use ipairs e.g.:

`for _, normValue in ipairs{1, 2, math.random()} do`